dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -640,6 +641,18 @@ const borderStandfirst = (format: ArticleFormat): string => {
 	return border.secondary;
 };
 
+const borderSecondary = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.LiveBlog) return border.secondary;
+	if (format.design === ArticleDesign.DeadBlog) return border.secondary;
+	return border.secondary;
+};
+
+const sportBorderTop = (format: ArticleFormat): string => {
+	if (format.design === ArticleDesign.LiveBlog) return palette.sport[300];
+	if (format.design === ArticleDesign.DeadBlog) return palette.sport[300];
+	return palette.sport[300];
```

Comment:
I think in this case we can simply just return `palette.sport[300];` - there's no need to use format to make a choice.

So why use `palette`? Because we want to centralise colour choice to make things like edits or maybe dark mode easier later on.